tests/validation/tests/v3_api/test_custom_host_reg.py

```diff
@@ -44,22 +45,13 @@ def test_delete_keypair():
 
 
 def test_deploy_rancher_server():
-    if "v2.5" in RANCHER_SERVER_VERSION or \
-            "master" in RANCHER_SERVER_VERSION or \
-            "v2.6" in RANCHER_SERVER_VERSION or \
-            "v2.7" in RANCHER_SERVER_VERSION or \
-            "v2.8" in RANCHER_SERVER_VERSION:
-        RANCHER_SERVER_CMD = \
-            'sudo docker run -d --privileged --name="rancher-server" ' \
-            '--restart=unless-stopped -p 80:80 -p 443:443  ' \
-            '-e CATTLE_BOOTSTRAP_PASSWORD="{}" ' \
-            'rancher/rancher'.format(ADMIN_PASSWORD)
-    else:
-        RANCHER_SERVER_CMD = \
-            'sudo docker run -d --name="rancher-server" ' \
-            '--restart=unless-stopped -p 80:80 -p 443:443  ' \
-            'rancher/rancher'
-    RANCHER_SERVER_CMD += ":" + RANCHER_SERVER_VERSION + " --trace"
+
+    RANCHER_SERVER_CMD = \
+        'sudo docker run -d --privileged --name="rancher-server" ' \
+        '--restart=unless-stopped -p 80:80 -p 443:443  ' \
+        '-e CATTLE_BOOTSTRAP_PASSWORD="{}" '.format(ADMIN_PASSWORD)
```

Comment:
why not just add the repo command here since we're already formatting the string. It was added to the string in the past due to the if condition, but now that we don't have it, I think it reads better to just format everything in this block. 

I also think that only the repo name, not the package name, should be configurable <repo>/<package>:<version> since this should always be deploying some version of rancher

Reply:
Adding the repo/image and server version might add more confusion as the command for this when we pass the RANCHER_REPOSITORY_IMAGE and RANCHER_SERVER_VERSION will be as follows:

```
RANCHER_SERVER_CMD = \
    (
    'sudo docker run -d --privileged --name="rancher-server" '
    '--restart=unless-stopped -p 80:80 -p 443:443  '
    '-e CATTLE_BOOTSTRAP_PASSWORD="{}" '
    '{}:{}'
).format(ADMIN_PASSWORD, RANCHER_REPOSITORY_IMAGE,RANCHER_SERVER_VERSION).
```
So, I added it in an extra line. 

> only the repo name, not the package name, should be configurable /: since this should always be deploying some version of rancher

I'm not sure I understand this

Reply:
spoke offline, and it seems for non-production rancher, the package name `rancher` can vary as well. I will approve based on this explanation.